src/components/map/Map.tsx

```diff
@@ -157,131 +144,264 @@ export const Map = ({
     };
   }
 
+  useEffect(() => {
+    if (clusteringEnabled) {
+      const index = new Supercluster({
+        radius: clusterDistance,
+        maxZoom: 20
```

Comment:
ha! the 20 is also used in multiple places, right? so please create a const for it and use it everywhere.

Reply:
its only used once, as the 15 is, just 14 is used twice. should it nevertheless be a const?

Reply:
yes please all of those values distributed across the file to be able to see them at one spot.

Reply:
will be adjusted in the upcoming commit.